### ISSUE TITLE
Radial Menu is broken

### ISSUE THREAD

**Issue by InfernoFalcon** [TheCyberBrick/The-Betweenlands#689]
#### Betweenlands version:
3.3.11

#### Forge version:
14.23.4.2705

#### Singleplayer or Multiplayer:
Singleplayer

#### Installed mods:
https://docs.google.com/document/d/1Pc-21dFjzXeKBgLVH-_JXepFJwlMMFHX5qKDtqUBb-Y/edit

#### How to reproduce:

This glitch always occurs, no steps needed to replicate the issue.

#### Expected result:
The radial menu would appear and I would be able to equip/unequip items such as amulets and the lurker skin pouch.

#### Actual result:
The radial menu prevents me from looking around and my cursor is visible but the interface is invisible. I an unable to equip/unequip and items and any items that are already equipped are stuck.
I can still open with a keybind however.

**Comment by GoryMoon** [comment]
Can you include a screenshot of the screen when it occurs? Also the link for the used mods requires read access.

**Comment by InfernoFalcon** [comment]
Here's the link to the video: https://gfycat.com/GeneralSlightBasilisk  Btw, you should be able to access the modlist now.

**Comment by TheCyberBrick** [comment]
Hm, no idea how this would happen. If you're still having this problem please send me your save and all installed mods so I can try to figure out what's happening.

**Comment by nataliat1026** [comment]
I actually came across this post while trying to find a solution to the exact same issue I've been having. It occurs both in single player games and on a server my friend and I are playing on and I've made a note of mods that overlap with those listed by the person in the original ticket. Short of removing every mod (we are playing with a lot, heads up) and adding in a few at a time and testing it that way, we're not sure what to do either. With the current issue, I think the only way to get items from the mod off is to kill myself in game. Any help would be so greatly appreciated...
